fadingpromises/setup-fadingpromises.tp2

```diff
@@ -45,6 +46,7 @@ LANGUAGE ~Deutsch~  ~german~  ~fadingpromises/german/setup-fadingpromises.tra~
 LANGUAGE ~Italiano~ ~italian~  ~fadingpromises/italian/setup-fadingpromises.tra~
 LANGUAGE ~Russian~ ~russian~  ~fadingpromises/russian/setup-fadingpromises.tra~
 LANGUAGE ~Francais~ ~french~  ~fadingpromises/french/setup-fadingpromises.tra~
+LANGUAGE ~Simplified Chinese~ ~schinese~  ~fadingpromises/schinese/setup-%WEIDU_OS%.tra~
```

Comment:
Hello @Lzw104522773
Could you please add only a single file for installation prompts purposes? As per our previous discussion, it could be "install.tra"